Assets/Scripts/playerController.cs

```diff
@@ -7,51 +7,63 @@ public class playerController : MonoBehaviour
      public Animator Animator;
     [SerializeField]
     float Jump_Power;
+    [SerializeField]
```

Comment:
Follow the C# naming convention when writing variables.

Reply:
Okay sure